hw/top_earlgrey/lint/top_earlgrey_lint_cfgs.hjson

```diff
@@ -0,0 +1,140 @@
+                  rel_path: "hw/ip/rv_dm/lint"
+             },
+             {    name: rv_plic_example
+                  fusesoc_core: lowrisc:ip:rv_plic_example
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/rv_plic_example/lint"
+             },
+             {    name: rv_timer
+                  fusesoc_core: lowrisc:ip:rv_timer
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/rv_timer/lint"
+             },
+             {    name: spi_device
+                  fusesoc_core: lowrisc:ip:spi_device
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/spi_device/lint"
+             },
+             {    name: uart
+                  fusesoc_core: lowrisc:ip:uart
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/uart/lint"
+             },
+             {    name: usbdev
+                  fusesoc_core: lowrisc:ip:usbdev
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/usbdev/lint"
+             },
+             {    name: usb_fs_nb_pe
+                  fusesoc_core: lowrisc:ip:usb_fs_nb_pe
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/usb_fs_nb_pe/lint"
+             },
+             {    name: usbuart
+                  fusesoc_core: lowrisc:ip:usbuart
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/usbuart/lint"
+             },
+             {    name: socket_1n
+                  fusesoc_core: lowrisc:tlul:socket_1n
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/tlul/socket_1n/lint"
+             },
+             {    name: socket_m1
+                  fusesoc_core: lowrisc:tlul:socket_m1
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/tlul/socket_m1/lint"
+             },
+             {    name: adapter_reg
+                  fusesoc_core: lowrisc:tlul:adapter_reg
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/tlul/adapter_reg/lint"
+             },
+             {    name: adapter_sram
+                  fusesoc_core: lowrisc:tlul:adapter_sram
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/tlul/adapter_sram/lint"
+             },
+             {    name: sram2tlul
+                  fusesoc_core: lowrisc:tlul:sram2tlul
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
+                  rel_path: "hw/tlul/sram2tlul/lint"
```

Comment:
Should the `rel_path` for these be `hw/ip/...`?

Reply:
I structured the relative paths here such that they correspond to the fusesoc cores for consistency... but I can introduce another level of hierarchy if needed

Reply:
The fusesoc core files are indeed in `hw/ip/tlul/` area. There is no `hw/tlul` directory.

Reply:
that is true, I was referring to the naming of the cores - i.e. `:tlul:sram2tlul` instead of `:ip:tlul:sram2tlul`. but let's just align with the directory structure then.

Reply:
Aah I see.